src/main/java/com/github/martapietka/ppm2pbm/PpmToPbmConverter.java

```diff
@@ -24,5 +23,23 @@ public static void convert(InputStream inputStream, OutputStream outputStream) t
         while (inputStream.read() > 0x1F) {
             // skip comment
         }
+
+        byte[] newHeader = {header[0], 0x31, header[2]};
+
+        int width = ByteReader.convertBytesToInt(inputStream);
+        int height = ByteReader.convertBytesToInt(inputStream);
+
+        String newWidth = Integer.toString(width);
+        newWidth.getBytes(StandardCharsets.UTF_8);
+
+        String newHeight = Integer.toString(height);
+        newHeight.getBytes(StandardCharsets.UTF_8);
+
+        PrintStream printStream = new PrintStream(outputStream);
+
+        printStream.write(newHeader);
+        printStream.write(newWidth.getBytes());
+        printStream.write(" ".getBytes());
```

Comment:
Skoro to jeden znak, możesz iść na skróty i użyć po prostu literału bajtowego, spacja to bodajże `0x20`?